docs/quick-start/site-generator.mdx

```diff
@@ -17,9 +17,7 @@ Use the Mosaic Site generator to generate a site and add an example source.
 The site generator expects to be run in a monorepo. If you are trying to set up a site in an empty directory, you will first need to run `yarn init` to create a `package.json` file.
 
 ```
-> yarn add @jpmorganchase/mosaic-create-site -W
-> yarn mosaic-create-site init
-> yarn mosaic-create-site create -o my-sample-site
+> npx @jpmorganchase/mosaic-create-site -o my-sample-site
```

Comment:
is it not still mosaic-create-site *create* -o my-sample-site? 
I can't see any changes that mean we can remove create